examples/aws-lambda-deployment/bal_build.out

```diff
@@ -0,0 +1,12 @@
+$ bal build
+Compiling source
+        wso2/aws_lambda_deployment:0.1.0
+
+Generating executable
+        @awslambda:Function: echo, ctxinfo, notifyS3
+
+        Run the following command to deploy each Ballerina AWS Lambda function:
```

Comment:
```suggestion
        Run the following command to deploy each Ballerina AWS Lambda function:
```
These instructions should be taken out of the `.out` file. The `.out` file should contact only the output. Need to check and correct all the occurrences.

Reply:
This instruction set is printed from the compiler itself as an output. Its oky to keep it here right?